extension/src/popup/styles.scss

```diff
@@ -1,3 +1,7 @@
 :root {
   --z-index--settings: 10;
+
+  // popup dimensions
+  --popup--width: 456px;
```

Comment:
❓ where is 456 from? in figma it says 360px?

Reply:
Ah good catch - `456` is the existing width of the popup in v1. I'll change to `360`